pkg/render/fluentd.go

```diff
@@ -90,6 +96,10 @@ const (
 
 	PacketCaptureAPIRole        = "packetcapture-api-role"
 	PacketCaptureAPIRoleBinding = "packetcapture-api-role-binding"
+
+	// Verification Mode for syslog forwarding.
```

Comment:
We can get rid of these values and also the verify_mode, so the default (secure) verification is always used.

Reply:
Updated